package.json

```diff
@@ -55,6 +55,7 @@
     "react-native-securerandom": "0.1.1",
     "react-native-splash-screen": "3.0.6",
     "react-native-svg": "6.3.1",
+    "react-native-swipeable": "^0.6.0",
```

Comment:
hey @ConnorChristie we're trying to do not use external libs for security reasons, could you take a look at https://github.com/status-im/status-react/blob/6fc03fc217f3f8cfa9debf4fbf9307a586a3f605/src/status_im/ui/screens/home/views.cljs#L49 maybe you could use this approach

Reply:
I don't think I would be able to get to that in this PR. I have spent way too much time on these PRs for what they're worth.

Reply:
There is quite a lot of code handling that single swipe to delete which would either need to be refactored to work with network settings or just copy, pasted, and adapted to work with network settings. (Quite a lot of work)

The concern regarding security is valid but this package is well worth it and can easily be audited since it contains <700 lines of code and one dep which has 2 million downloads.

I would ask that you reconsider keeping this module for ease of use and allowing the code to focus on the business logic rather than implementing custom swipe animations in addition.

Reply:
We carefully consider introducing extra dependencies as it introduces security risks. That's the reason we have our own code for swipe support.

Reply:
Yes, I understand that but as I mentioned, the code for that package is very small with only one dep of its own. This issue has also been opened (#4247) regarding the not so good feel of the current implementation.

I believe it would be worth the security audit of the external dep over trying to improve the current custom implementation.

Reply:
One might also even suggest forking that swipable package into status and removing the external dependency

Reply:
It might be something that will be reconsidered later but not now and not in the context of this PR (swipe to delete is not the primary goal here).
Besides it would mean 2 different UX for swipe to delete.

Reply:
So without duplicating the current swipe to delete code for this screen or using this external library, I would then suggest a simple delete button be shown on the Network details page if it is a custom network. Does that sound good?

Reply:
Sounds good. Or even not allowing to delete for now would be good if simpler.